test/nightly/azureResourceMongoDB.test.ts

```diff
@@ -63,6 +63,18 @@ suite('MongoDB action', async function (this: ISuiteCallbackContext): Promise<vo
         assert.ok(collection);
     });
 
+    test('Delete account', async () => {
+        const createAccount: CosmosDBManagementModels.DatabaseAccount = await client.databaseAccounts.get(resourceGroupName, accountName);
```

Comment:
Could you rename this to just `mongoAccount`? Since this isn't actually creating an account.